### DISCUSSION TITLE
[Bug]: litellm start segment fault

### DISCUSSION THREAD

**Discussion by hanrui1sensetime** [BerriAI/litellm#3778] [Q&A]
### What happened?

We use OpenEuler 22.03 System(Like CentOS 7).
We use pip install litellm[proxy]==1.37.0
When we start litellm by command: litellm, it crashed, and the log only `Segmentation fault`. 
How to solve it? Thx!

### Relevant log output

`Segmentation fault`

**Comment by ishaan-jaff** [comment]
<img width="1280" alt="Screenshot 2024-05-21 at 10 45 01 PM" src="https://github.com/BerriAI/litellm/assets/29436595/289394e4-e527-4371-9bc9-3a960b606384">


works fine for me

**Reply by ishaan-jaff** [reply]
converted to an issue since this looks isolated to your env @hanrui1sensetime 

can you send the terminal logs when you do this

**Reply by hanrui1sensetime** [reply]
![3778](https://github.com/BerriAI/litellm/assets/83800577/14d1afd0-2dd7-4e92-bca5-e09959c4dd70)
